sign/sign.go

```diff
@@ -128,7 +128,37 @@ func (s *Signer) SignImage(reference string) (*SignedObject, error) {
 func (s *Signer) SignFile(path string) (*SignedObject, error) {
 	s.log.Infof("Signing file path: %s", path)
 
-	// TODO: unimplemented
+	os.Setenv("COSIGN_EXPERIMENTAL", "true")
+	defer os.Setenv("COSIGN_EXPERIMENTAL", "")
```

Comment:
I think we should use `s.impl.Setenv` instead (see SignImage function as an example).

Reply:
Done!